cli/config/configfile/file.go

```diff
@@ -41,6 +41,7 @@ type ConfigFile struct {
 	CLIPluginsExtraDirs  []string                     `json:"cliPluginsExtraDirs,omitempty"`
 	Plugins              map[string]map[string]string `json:"plugins,omitempty"`
 	Aliases              map[string]string            `json:"aliases,omitempty"`
+	AutoRemove           bool                         `json:"autoRemove,omitempty"`
```

Comment:
Considering the above, would `DefaultAutoRemove` be a better name for this?

```suggestion
	DefaultAutoRemove           bool                         `json:"defaultAutoRemove,omitempty"`
```

Reply:
Having said that i think it'd be worth thinking about how to model other CLI flags in config.json – i'm not wild about the idea of a special "defaultXYZ" struct because of the idiom i refer to above, but on the other hand i would also understand not wanting to clutter the `ConfigFile` struct 🤔 Up to you!